classy-prelude/ClassyPrelude.hs

```diff
@@ -175,8 +208,11 @@ import Data.Functor
 import Control.Exception (assert)
 import Control.Exception.Enclosed
 import Control.Monad (when, unless, void, liftM, ap, forever, join, replicateM_, guard, MonadPlus (..), (=<<), (>=>), (<=<), liftM2, liftM3, liftM4, liftM5)
+import Control.Concurrent.Lifted
```

Comment:
Still, I'd rather go for consistency with other modules with conflicting identifiers (like `Control.Concurrent.STM`). It also reduces the maintenance burden in the future, since new upstream identifiers are automatically added.